### PR TITLE
Feat/basics

### PR DESCRIPTION
**Pull request by vahidabbasi**

### CODE REVIEW

**Comment by vahidabbasi** [comment]
if you want to check:
https://build-nfcnkqpisy.now.sh/

File: .idea/codeStyles/Project.xml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+        <option name="OBJECT_CASE" value="preserve" />
+      </case-options>
+      <formatting-settings enabled="false">
+        <option name="STATEMENT_SPACING" value="one_line" />
+        <option name="CLAUSE_CHOP_DOWN" value="chop_down_if_statement_long" />
+        <option name="ITERATION_ELEMENTS_WRAPPING" value="chop_down_if_not_single" />
+      </formatting-settings>
+    </DBN-SQL>
+  </code_scheme>
+</component>
```
Comment (iamstarkov) [inline .idea/codeStyles/Project.xml]: you need to add `.idea` to `.gitignore`

Comment (iamstarkov) [reply .idea/codeStyles/Project.xml]: and remove these files from this pull-request

File: src/index.css [hunk 2]
```diff
@@ -2,4 +2,22 @@ body {
   margin: 0;
   padding: 0;
   font-family: sans-serif;
+  margin-top: 40px;
```
Comment (iamstarkov) [inline src/index.css]: you don't need this line, I had it in my css only because projector in a room sucks

**Comment by iamstarkov** [comment]
dont forget to delete branches after you merged 'em

**Pull request merged**